frontend/nginx.conf

```diff
@@ -16,11 +16,22 @@ http {
 
     access_log /var/log/nginx/nginx-frontend.log;
 
+    gzip on;
+    gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
+    gzip_min_length 256;
```

Comment:
Is there a reason to not use the default value of 20?

Reply:
For file sizes that small, the extra cpu overhead is usually not worth the size reduction